docs/apache-airflow/authoring-and-scheduling/timetable.rst

```diff
@@ -179,34 +179,15 @@ first) event for the data interval, otherwise manual runs will run with a ``data
 
 .. _dataset-timetable-section:
 
-DatasetTimetable
-^^^^^^^^^^^^^^^^
+Integrate conditional dataset with Time-Based Scheduling
```

Comment:
Can you enhance it better? 
may be "Dataset event based scheduling with time based scheduling"